vmdb/app/models/configuration_profile_foreman.rb

```diff
@@ -19,7 +19,8 @@ class ConfigurationProfileForeman < ConfigurationProfile
   belongs_to :customization_script_ptable
   belongs_to :customization_script_medium
   belongs_to :operating_system_flavor
-  virtual_has_many :configuration_tags,  :class_name => 'ConfigurationTag'
+  virtual_has_many :configuration_tags,  :class_name => 'ConfigurationTag', :uses => [:direct_configuration_tags, :parent]
+  virtual_has_one :configuration_architecture, :class_name => 'ConfigurationArchitecture', :uses => :configuration_tags
```

Comment:
1. not sure if `configuration_tags` is still needed.
Think `configuration_tags` is now an actual relation

2. should we create virtual attributes for all these attributes?
I'll create a pr.